File: PYTHON/updateTicket.py
import requests,json

auth_token="59550a0e2b1a864a31bef962363e029f" #YOUR_AUTH_TOKEN
org_id="652853630" #YOUR_ORGANISATION_ID

ticket_id="215666000000137001"; #TICKET_ID_TO_UPDATE

ticket_data={
        "subject":"This sis python changed 2 sample ticket",
        "priority":"medium",
        "description":"Some changed description"
}

headers={
    "Authorization":auth_token,
    "orgId":org_id,
    "contentType": "application/json; charset=utf-8"
}

if ticket_id:

    request=requests.patch('https://desk.zoho.com/api/v1/tickets/'+ticket_id, headers=headers,data=json.dumps(ticket_data))

    if request.status_code == 200:
        print "Request Successful,Response:"
        print request.content
    else:
        print "Request not successful,Response code ",request.status_code," \nResponse : ",request.content;

else:
    print "Ticket ID is missing"

File: C Sharp/updateTicket.cs
using System;
using System.Net;
using System.Text;
using System.IO;

class UpdateTicket
{
    public static void Main(string[] args)
    {
        string auth_token = "59550a0e2b1a864a31bef962363e029f"; // YOUR_AUTH_TOKEN
        string org_id = "652853630"; //YOUR_ORG_ID

        string ticket_id = "215666000000198001";

        string ticket_data = "{\"subject\" : \" from C# 2 Updated Welcome to Zoho Support. Youve got a sample Request!\"}";

        if (ticket_id != "")
        {

            HttpWebRequest request = (HttpWebRequest)WebRequest.Create("https://desk.zoho.com/api/v1/tickets/"+ticket_id);
            request.ContentType = "application/json";
            request.Method = "PATCH";
            request.Headers["Authorization"] = auth_token;
            request.Headers["orgId"] = org_id;

            byte[] jsondata = Encoding.UTF8.GetBytes(ticket_data);
            request.ContentLength = jsondata.Length;

            Stream requestStream = request.GetRequestStream();
            requestStream.Write(jsondata, 0, jsondata.Length);
            requestStream.Close();

            try
            {
                HttpWebResponse response = (HttpWebResponse)request.GetResponse();
                Stream receiveStream = response.GetResponseStream();
                StreamReader readStream = new StreamReader(receiveStream, Encoding.UTF8);
                Console.WriteLine("Response Code : " + (int)response.StatusCode);
                if ((int)response.StatusCode == 200)
                {
                    Console.WriteLine("Ticket Successfully Updated ");
                    Console.WriteLine(readStream.ReadToEnd());
                    
                }
                response.Close();
                readStream.Close();

            }
            catch (WebException e)
            {
                Console.WriteLine("Request failed, Response Code : " + (int)((HttpWebResponse)e.Response).StatusCode + " \nResponse Message : ");
                var message = new StreamReader(e.Response.GetResponseStream()).ReadToEnd();
                Console.WriteLine(message);
            }

		}
		else
		{
			Console.WriteLine("ERROR : Ticket ID is missing");
		}

    }
}
